.github/workflows/canary-integration-test.yml

```diff
@@ -1307,7 +1312,6 @@ jobs:
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
 
-
```

Comment:
why these are removed

Reply:
Hmm, just noticed this. I guess due to some lint plugin in vscode these lines were removed, shall, I remove them or let it be? Anyway, seems like these were extra lines 🤔